Project/Src/Gui/FoldMargin.cs

```diff
@@ -110,7 +110,7 @@ bool SelectedFoldingFrom(IEnumerable<FoldMarker> list)
 
                 DrawFoldMarker(
                     g, new RectangleF(
-                        drawingRectangle.X + (drawingRectangle.Width - foldMarkerSize)/2,
+                        drawingRectangle.X + (drawingRectangle.Width - foldMarkerSize)/2f,
```

Comment:
This may have been intentional, to keep pixel rendering crisp on whole pixel boundaries, rather than fuzzy when rendered to subpixels.